django/contrib/staticfiles/handlers.py

```diff
@@ -53,12 +53,10 @@ def serve(self, request):
     def get_response(self, request):
         from django.http import Http404
```

Comment:
Could you possibly move this import to the top in a following commit?

Reply:
I can... 🙂 Let me try that.